node/node.go

```diff
@@ -48,6 +48,11 @@ const (
 	NodeLeader                         // Node is the leader of some shard.
 )
 
+const (
+	// TxPoolLimit is the limit of transaction pool.
+	TxPoolLimit = 20000
```

Comment:
Can we make it bigger?  what's the proper number?

Reply:
one tx is around 250-500 bytes. we should be able to do 200000

Reply:
1/ 20k TPS is a record for one shard. i don't think we would expect that.
2/ the goal of this is to find the culprit of OOM - If this really reaches 20k there is something wrong for example the finality is too long which, going forward, we need to look into.

Reply:
discord discussed